docs/Prerequisites.md

```diff
@@ -2,17 +2,17 @@
 
 ## Essentials
 
-1. Download and install the latest version of **[Unity 2020.2](https://unity3d.com/get-unity/download)**   
-2. Use the commands below to clone the [SynthDet repository](https://github.com/Unity-Technologies/SynthDet) and initialize the LFS (large file storage) submodules:
-```bash
+1. Download and install the latest version of **[Unity 2020.3](https://unity3d.com/get-unity/download)**. When prompted during installation, make sure the **Linux Build Support (Mono)** module is checked.   
+2. Use the commands below to clone the SynthDet repository and initialize the LFS (large file storage) submodules:
+```
 git lfs install
 git clone https://github.com/Unity-Technologies/SynthDet
 cd SynthDet
 git submodule update --init --recursive
```

Comment:
There are no longer submodules in this repository, so this can be simplified

Reply:
good call!